actors/builtin/miner/miner_actor.go

```diff
@@ -582,6 +585,13 @@ func (a Actor) ProveCommitSector(rt Runtime, params *ProveCommitSectorParams) *a
 func (a Actor) ConfirmSectorProofsValid(rt Runtime, params *builtin.ConfirmSectorProofsParams) *adt.EmptyValue {
 	rt.ValidateImmediateCallerIs(builtin.StoragePowerActorAddr)
 
+	if len(params.Sectors) > power.MaxMinerProveCommitsPerEpoch {
+		// XXX: log only?
```

Comment:
This shouldn't be possible. Personally, I'd remove this or log, but I wanted to bring it up.